packages/core/src/rules/__tests__/no-unresolved-refs.test.ts

```diff
@@ -196,4 +196,33 @@ describe('oas3 boolean-parameter-prefixes', () => {
 
     expect(replaceSourceWithRef(results, __dirname)).toMatchInlineSnapshot(`Array []`);
   });
+
+  it('should not report on nested refs inside specification extensions', async () => {
+    const document = parseYamlToDocument(
+      outdent`
+        openapi: 3.0.0
```

Comment:
Could you write also a test for 3.1? Just to be sure :)

Reply:
Yeah, of course.